assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
+    school = models.CharField(max_length=256)
```

Comment:
Why is this 256? 255 is the max value for a char field in MySQL...

Also note, if a char field is ever to be a indexed field, please use 191 for the length as that is compliant with utf8mb4 collation in MySQL (aka. it would support 4 byte unicode chars, for instance some emojis)